assets/components/src/autocomplete-tokenfield/style.scss

```diff
@@ -0,0 +1,32 @@
+.autocomplete-tokenfield {
```

Comment:
The classname should also be updated here, and on line 22

Reply:
Doh!